docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:12.3
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: "${DB_PASSWORD:-notinproduction}"
```

Comment:
This is where we're actually telling the database what its default password is. This is also configurable. It will look for an env var called `DB_PASSWORD` and, if that's not set, will then use the string `notinproduction`.

Reply:
Then why the need to put a default above as well? This file is also not included in .gitignore